### ISSUE TITLE
fix `write_csv` warning

### ISSUE THREAD

**Issue by joelgombin** [joelgombin/banR#34]
```
The `path` argument of `write_csv()` is deprecated as of readr 1.4.0.
Please use the `file` argument instead.
This warning is displayed once every 8 hours.
Call `lifecycle::last_warnings()` to see where this warning was generated.
```

**Comment by JMPivette** [comment]
For information, this issue is already solved in my PR #32 :-)

**Comment by pachevalier** [comment]
Ok, I've merged the issue. Thanks
